### PR TITLE
Patch/onedr0p

### PR DESCRIPTION
**Pull request by onedr0p**
#### Database Migration
NO

#674 
#697

### CODE REVIEW

**Comment by onedr0p** [comment]
@galli-leo please take a look at commit https://github.com/Radarr/Radarr/pull/716/commits/3bf9f45c5046d53deb3faa643ef54c850a40c443

And let me know if it looks alright.

File: src/NzbDrone.Core/Housekeeping/Housekeepers/CleanupOrphanedPendingReleases.cs [hunk 1]
```diff
@@ -18,9 +18,9 @@ public void Clean()
             mapper.ExecuteNonQuery(@"DELETE FROM PendingReleases
                                      WHERE Id IN (
                                      SELECT PendingReleases.Id FROM PendingReleases
-                                     LEFT OUTER JOIN Series
-                                     ON PendingReleases.SeriesId = Series.Id
-                                     WHERE Series.Id IS NULL)");
+                                     LEFT OUTER JOIN Movies
+                                     ON PendingReleases.MovieId = MovieId.Id
+                                     WHERE MovieId.Id IS NULL)");
```
Comment (galli-leo) [inline src/NzbDrone.Core/Housekeeping/Housekeepers/CleanupOrphanedPendingReleases.cs]: This needs to be Movies.Id IIRC. Please run the Housekeeping commands from System -> Tasks and see if it completes sucessfully.

Comment (onedr0p) [inline src/NzbDrone.Core/Housekeeping/Housekeepers/CleanupOrphanedPendingReleases.cs]: Damn, it's late here :P thanks for seeing that.

Comment (onedr0p) [inline src/NzbDrone.Core/Housekeeping/Housekeepers/CleanupOrphanedPendingReleases.cs]: Should be good now.

**Pull request merged**